src/extension/ipc.cc

```diff
@@ -32,12 +32,7 @@ bool sapi_ipc_router_map (
     auto router,
     auto reply
   ) mutable {
-    auto msg = SSC::IPC::Message(
-      message.uri,
-      true, // decode parameter values AOT in `message.uri`
-      message.buffer.bytes,
-      message.buffer.size
-    );
+    auto msg = SSC::IPC::Message(message);
```

Comment:
We wanted an AOT decode so the `sapi_ipc_message_get()` calls get a decoded value

Reply:
I've guaranteed that `Router::invoke` receives an `IPC::Message` with decoded values in other parts of this PR.

https://github.com/aleclarson/socket/blob/b9449443918685f6b2d15cca18ad62efd182ed06/src/ipc/bridge.cc#L2728-L2736

When `Router::invoke` is called with a `uri` string, values are always decoded (is this a problem?). When `Router::invoke` is called with a `IPC::Message` instead, it's assumed that the message has decoded values, which I've checked is true anywhere this overload is used as of now.